testing/scenario_app/lib/src/scenarios.dart

```diff
@@ -0,0 +1,60 @@
+import 'send_text_focus_semantics.dart';
+import 'touches_scenario.dart';
+
+Map<String, Scenario> _scenarios = <String, Scenario>{
+  'animated_color_square': AnimatedColorSquareScenario(window),
+  'platform_view': PlatformViewScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_no_overlay_intersection': PlatformViewNoOverlayIntersectionScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_partial_intersection': PlatformViewPartialIntersectionScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_two_intersecting_overlays': PlatformViewTwoIntersectingOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_one_overlay_two_intersecting_overlays': PlatformViewOneOverlayTwoIntersectingOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_multiple_without_overlays': MultiPlatformViewWithoutOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_max_overlays': PlatformViewMaxOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
+  'platform_view_cliprect': PlatformViewClipRectScenario(window, 'PlatformViewClipRect', id: 1),
+  'platform_view_cliprrect': PlatformViewClipRRectScenario(window, 'PlatformViewClipRRect', id: 2),
+  'platform_view_clippath': PlatformViewClipPathScenario(window, 'PlatformViewClipPath', id: 3),
+  'platform_view_transform': PlatformViewTransformScenario(window, 'PlatformViewTransform', id: 4),
+  'platform_view_opacity': PlatformViewOpacityScenario(window, 'PlatformViewOpacity', id: 5),
+  'platform_view_multiple': MultiPlatformViewScenario(window, firstId: 6, secondId: 7),
+  'platform_view_multiple_background_foreground': MultiPlatformViewBackgroundForegroundScenario(window, firstId: 8, secondId: 9),
+  'poppable_screen': PoppableScreenScenario(window),
+  'platform_view_rotate': PlatformViewScenario(window, 'Rotate Platform View', id: 10),
+  'platform_view_gesture_reject_eager': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: false),
+  'platform_view_gesture_accept': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: true),
+  'platform_view_gesture_reject_after_touches_ended': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: false, rejectUntilTouchesEnded: true),
+  'tap_status_bar': TouchesScenario(window),
+  'text_semantics_focus': SendTextFocusScemantics(window),
+};
+
+Map<String, dynamic> _currentScenario = <String, dynamic>{
+  'name': 'animated_color_square',
+};
+
+/// Loads an scenario.
+/// The map must contain a `name` entry, which equals to the name of the scenario.
+void loadScenario(Map<String, dynamic> scenario) {
+  assert(_scenarios[scenario['name']] != null);
+  _currentScenario = scenario;
+  window.scheduleFrame();
+}
+
+/// Gets the loaded [Scenario].
+Scenario getCurrentScenario() {
```

Comment:
nit: `Scenario get currentScenario {`. This isn'tdoing any special expensive logic or state mutation.

Reply:
Done.